Sources/hostmgr/commands/vm/VMStop.swift

```diff
@@ -19,12 +19,17 @@ struct VMStopCommand: ParsableCommand {
     var all: Bool = false
 
     func run() throws {
-        try all ? shutdownAll() : virtualMachine?.shutdown()
-    }
 
-    private func shutdownAll() throws {
-        try Parallels()
-            .lookupRunningVMs()
-            .forEach { try $0.shutdown(immediately: immediately) }
+        if all {
+            try libhostmgr.stopAllRunningVMs(immediately: self.immediately)
+            Console.exit()
+        }
+
+        // In the case of invalid input like this, show the help text
+        guard let identifier = self.identifier else {
```

Comment:
Nitpick. This could become shorter thanks to SE-0345, see https://github.com/wordpress-mobile/WordPress-iOS/issues/19549

```suggestion
        guard let identifier else {
```

Reply:
Ah nice! I've been using it for `if let`, but I guess it makes sense this would also work for `guard let`!

Cleaned up in 3c220543f430a0fa4b9f27d11c47f2ba256627ea